spark/src/test/java/com/netflix/iceberg/spark/source/TestParquetWrite.java

```diff
@@ -71,7 +72,9 @@ public static void stopSpark() {
   public void testBasicWrite() throws IOException {
     File parent = temp.newFolder("parquet");
     File location = new File(parent, "test");
+    File dataLocation = new File(parent, "test-data");
```

Comment:
This test should not be modified because it tests the default behavior. There should be an additional test for behavior with the new property. I'd also like to see a test that sets the property on the table instead of in the write options, and another one where both are set and the write option takes precedence.

Also, I think this should probably go into a more general test suite. This one is specific to Parquet, but you're testing the behavior for any file format.